crates/rolldown/tests/esbuild/dce/preserve_directives_minify_bundle/artifacts.snap

```diff
@@ -1,27 +1,24 @@
 ---
 source: crates/rolldown_testing/src/integration_test.rs
-snapshot_kind: text
 ---
 # Assets
 
 ## entry.js
 
 ```js
+'use 1'
+'use 2'
+'use 3'
+
 (function() {
 
 "use strict";
 
 //#region nested.js
-"use A";
```

Comment:
What should happen to these?

Reply:
just align with esbuild, 
![image](https://github.com/user-attachments/assets/28e9f3a0-2168-4635-846d-f55fe485d804)